src/components/ReportsPage/HeatmapPage/Tables/LevelsDrivenTable.tsx

```diff
@@ -77,13 +84,6 @@ export const LevelsDrivenTable = ({
                   />
                 </TableCell>
               )}
-              {notGroupedByServices && (
```

Comment:
Is this change intentional? It doesn't match the PR or changelog description

Reply:
Yes, sorry, I moved this to just remove the unnecessary condition here.